client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -0,0 +1,636 @@
+import {
+	canCancelCampaign,
+	getAdvertisingDashboardPath,
+} from 'calypso/my-sites/promote-post-i2/utils';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import {
+	formatCents,
+	formatNumber,
+	getCampaignDurationFormatted,
+	getCampaignEstimatedImpressions,
+	getCampaignStatus,
+	getCampaignStatusBadgeColor,
+} from '../../utils';
+
+interface Props {
+	isLoading?: boolean;
+	siteId?: number;
+	campaign?: CampaignResponse;
+}
+
+const FlexibleSkeleton = () => {
+	return <div className="campaign-item-details__flexible-skeleton" />;
+};
+
+const getPostIdFromURN = ( targetUrn: string ) => {
+	if ( ! targetUrn.includes( ':' ) ) {
+		return;
+	}
+
+	const splitted = targetUrn.split( ':' );
+	if ( splitted.length >= 4 ) {
+		return splitted[ 4 ];
+	}
+};
+
+export default function CampaignItemDetails( props: Props ) {
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+
+	const translate = useTranslate();
+	const [ showDeleteDialog, setShowDeleteDialog ] = useState( false );
+	const [ showErrorDialog, setShowErrorDialog ] = useState( false );
+	const { cancelCampaign } = useCancelCampaignMutation( () => setShowErrorDialog( true ) );
+	const isSmallScreen = useBreakpoint( '<660px' );
+	const { campaign, isLoading, siteId } = props;
+	const campaignId = campaign?.campaign_id;
+
+	const {
+		audience_list,
+		content_config,
+		start_date,
+		end_date,
+		display_name,
+		creative_html,
+		width,
+		height,
+		status,
+		campaign_stats,
+		billing_data,
+		display_delivery_estimate = '',
+		target_urn,
+		delivery_percent,
+	} = campaign || {};
+
+	const {
+		impressions_total = 0,
+		clicks_total,
+		clickthrough_rate,
+		duration_days,
+		total_budget_left,
+		total_budget,
+		overall_spending,
+		visits_total = 0,
+		visits_organic,
+	} = campaign_stats || {};
+
+	const { card_name, payment_method, subtotal, credits, total } = billing_data || {};
+	const { title, clickUrl } = content_config || {};
+
+	const onClickPromote = useOpenPromoteWidget( {
+		keyValue: `post-${ getPostIdFromURN( target_urn || '' ) }`, // + campaignId,
+		entrypoint: 'promoted_posts-post_item',
+	} );
+
+	// Target block
+	const devicesList = audience_list ? audience_list[ 'devices' ] : '';
+	const countriesList = audience_list ? audience_list[ 'countries' ] : '';
+	const topicsList = audience_list ? audience_list[ 'topics' ] : '';
+	const OSsList = audience_list ? audience_list[ 'OSs' ] : '';
+
+	// Formatted labels
+	const ctrFormatted = clickthrough_rate ? `${ clickthrough_rate.toFixed( 2 ) }%` : '-';
+	const durationFormatted = getCampaignDurationFormatted( start_date, end_date );
+	const totalBudgetFormatted = `$${ formatCents( total_budget || 0 ) }`;
+	const totalBudgetLeftFormatted = `$${ formatCents( total_budget_left || 0 ) } ${ __( 'left' ) }`;
+	const overallSpendingFormatted = `$${ formatCents( overall_spending || 0 ) }`;
```

Comment:
I checked the field `overall_spending` in A8C DSP and think that it's not used there anymore.

In opposition, I've found that in API response we have a field `total_budget_used`. I assume this field should be used instead.

([source](https://github.tumblr.net/Tumblr/a8c-dsp/blob/trunk/dsp/src/api/routes/v1/api/campaigns.route.ts#L498))
<img width="480" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/eb9b0f6d-ecbf-49bc-bc97-d8756d813213">

Here's how a response from the API look like for me (there's no field `overall_spending` there):
<img width="501" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/507228c5-3be4-490b-83b6-4a75f7ac5ab1">